src/allmydata/web/private.py

```diff
@@ -61,7 +61,11 @@ def check(auth_token):
         pass
 
 
-@implementer(IToken)
+# Shoobx/mypy-zope#26
```

Comment:
Shoobx/mypy-zope#26 is closed now.  It sounds like there is some other resolution required here, perhaps having to do with using some type stubs for Twisted?  Alternatively, if you want to move `IToken` to `allmydata.interfaces` and that helps then feel free.

Either way, please update the comment to explain the pitfall and link to any outstanding tickets for work that might need to be done to address it.

Reply:
That's right - having type stubs for Twisted is the proper fix. Clarified in 51b0b201b.